### PR TITLE
[BEAM-1218] Removing some of the dataflow references.

### PR DESCRIPTION
**Pull request by aaltay**

### CODE REVIEW

**Comment by aaltay** [comment]
R: @davorbonaci Please review.

File: sdks/python/apache_beam/internal/auth.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-"""Dataflow credentials and authentication."""
```
Comment (davorbonaci) [inline sdks/python/apache_beam/internal/auth.py]: All, or at least parts of this, might be Dataflow / GCP specific.

File: sdks/python/apache_beam/io/bigquery.py [hunk 2]
```diff
@@ -37,13 +37,14 @@
...
 
 BigQuery sources can be used as main inputs or side inputs. A main input
-(common case) is expected to be massive and the Dataflow service will make sure
-it is split into manageable chunks and processed in parallel. Side inputs are
-expected to be small and will be read completely every time a ParDo DoFn gets
-executed. In the example below the lambda function implementing the DoFn for the
-Map transform will get on each call *one* row of the main table and *all* rows
-of the side table. The execution framework may use some caching techniques to
-share the side inputs between calls in order to avoid excessive reading::
+(common case) is expected to be massive and the execution framework will make
```
Comment (davorbonaci) [inline sdks/python/apache_beam/io/bigquery.py]: you are talking about what execution engine will do?
if you are using native source, it is runner-specific.
if not, then it is the source guaranteeing that.

Comment (aaltay) [inline sdks/python/apache_beam/io/bigquery.py]: Rewrote this part to be more generic, and easier for the user to understand.

File: sdks/python/apache_beam/internal/auth.py (hunk 1 above)
Comment (aaltay) [inline sdks/python/apache_beam/internal/auth.py]: Correct. Reverted this for now. I will tackle along with the future cleanups.

**Comment by aaltay** [comment]
Thank you @davorbonaci  please take another look.

**Comment by davorbonaci** [comment]
LGTM. Merged.

**Comment by aaltay** [comment]
Thank you.

**Pull request closed without merging**